cedar-policy/src/api.rs

```diff
@@ -263,6 +263,8 @@ impl Entity {
     }
 
     /// Parse an entity from an in-memory JSON value
+    /// If a schema is provided, it is handled identically to
+    /// [Entities](https://docs.rs/cedar-policy/latest/cedar_policy/struct.Entities.html#method.from_json_str)
```

Comment:
Why not just doc link?

Reply:
How do you auto-doc link to a specific method?

Reply:
`[Entities::from_json_str]` i think